ksa_compliance/output_models/e_invoice_output_model.py

```diff
@@ -38,7 +38,7 @@ def append_tax_details_into_item_lines(invoice_id, item_lines):
         item["tax_amount"] = tax_amount
         # TODO: In theory, net_amount includes the discount while amount doesn't. In practice, both have the same
```

Comment:
So, does net_amount differ from amount only when there's a line/item discount? If so, please update the comment and remove the TODO